### PR TITLE
New blogpost: Seven tips to succeed with DevOps - Norwegian

### PR DESCRIPTION
**Pull request by remiks**
Seven tips in Norwegian on how to succeed with DevOps. The PR is built locally and tested okay.

### CODE REVIEW

**Comment by remiks** [comment]
Dont merge this one. Just read QA and approve if its okay. I`ll merge it when ready to publish

**Review by alegri90 — APPROVED** [review]
Content reviewed - Looks good!
It builds fine as well 🤓
Leave in approved state, to allow the author to do the merge when it is publish ready for SoMe strategies and such 👍

**Pull request merged**
